packages/core/src/Cardano/types/StakePool/Cip6ExtendedStakePoolMetadata.ts

```diff
@@ -3,7 +3,7 @@ import { PoolIdHex } from './primitives';
 /**
  * Schema from:
  * https://raw.githubusercontent.com/cardano-foundation/CIPs/b93e77119e15d8763d7548a8a00c4cb7591714e4/CIP-0006/schema.json
- * This file was automatically generated by json-schema-to-typescript.
+ * This file was automatically generated by json-schema-to-typescript and then modified due to type names clashes.
```

Comment:
Can we not address the type name clashes in our definition? Modifying this auto-generated file is undesireable

Reply:
The only reason I modified this file is that here just one of the types is auto-generated with the common name `ExtendedStakePoolMetadataFields` so by adding a prefix `Cip6` and `AP` to the exported types the difference between the formats is easily visible to the eye.

Another way would be to keep the generated ts file as it is with `ExtendedStakePoolMetadataFields` (represent CIP-6 format) and then have another type `APExtendedStakePoolMetadata` (AdaPools)?

Reply:
> keep the generated ts file as it is with ExtendedStakePoolMetadataFields (represent CIP-6 format) and then have another type APExtendedStakePoolMetadata (AdaPools)?

Sounds reasonable

Reply:
The type changes are reverted back for this file as now we have types `ExtendedStakePoolMetadata`(CIP-6) and `APExtendedStakePoolMetadata` (AdaPools).

Updated with 5de0f21818716bc7d42d195ce28b17e7a349e890 (core) and 73fe4b5bb6df49342c35d925671757ca06f40f75 (cardano-services)